tests/unit/test_index.py

```diff
@@ -60,8 +60,9 @@ def check_caplog(caplog, expected_level, expected_message):
 
 @pytest.mark.parametrize('ignore_requires_python, expected', [
     (None, (
-        False, 'DEBUG',
-        "Link requires a different Python (3.6.5 not in: '== 3.6.4'): "
+        False, 'INFO',
+        "Package does not meet the Python version requirement. "
+        "Required version '== 3.6.4', Current version 3.6.5 "
         "https://example.com"
```

Comment:
The final link part looks weird to me, but I don’t have a good suggestion either 🤔

Reply:
The link is utilized for testing as I see here https://github.com/pypa/pip/blob/4775afb2d1f3ed73186257d652f81e0a78d5f7d4/tests/unit/test_index.py#L80-L82